examples/server-side-rendering/webpack.server.config.js

```diff
@@ -4,25 +4,24 @@ const path = require('path');
 const webpack = require('webpack');
 
 module.exports = {
+  mode: 'development',
```

Comment:
This is a workaround for this issue: https://github.com/angular/angular-cli/issues/8616

Reply:
It'd be nice to add this explanation in the code itself, so that next time we come across this line we know why it's here.

Reply:
yes will do!